data/json/recipes/weapon/magazines.json

```diff
@@ -120,18 +120,7 @@
     "time": "20 m",
     "autolearn": true,
     "tools": [
-      [
-        [ "acr", -1 ],
-        [ "ar15", -1 ],
-        [ "h&k416a5", -1 ],
-        [ "m249", -1 ],
-        [ "m27iar", -1 ],
-        [ "m4a1", -1 ],
-        [ "m16a4", -1 ],
-        [ "scar_l", -1 ],
-        [ "sig552", -1 ],
-        [ "surv_carbine_223", -1 ]
-      ],
+      [ [ "nato_assault_rifle", -1 ], [ "ar15", -1 ], [ "m249", -1 ], [ "m16a4", -1 ], [ "surv_carbine_223", -1 ] ],
```

Comment:
`m16a4` should be converted to `nato_assault_rifle` too.

Reply:
It does not have auto, it has burst. I converted the m16a3 to it because it has the same fire modes, but the m16a4 does not.
While it's not very different now, hopefully in the future it will matter.

Reply:
Could some of the other differences be accounted for as gunmods (e.g., a folding stock)? (Admittedly, they're likely not to be quite the same statistically - not coming built-in from the manufacturer.)

Reply:
It appears gunmods can change firing modes, so that is a route that we could go down.

Reply:
I've elected to just copy-from all non-differentiating information from the `nato_assault_rifle`, see the latest commit.